server/container/guice/cassandra-guice/src/main/java/org/apache/james/CassandraJamesServerMain.java

```diff
@@ -100,7 +99,6 @@
         new CassandraMailRepositoryModule(),
         new CassandraMetricsModule(),
         new BlobStoreAPIModule(),
-        new CassandraObjectStoreModule(),
```

Comment:
Cassandra-Guice should still be using the CassandraBlobStore.

We only change the behaviour in the new product...

Reply:
IMO there is no reason for changes in this file